### PR TITLE
doc: add a top level readme

### PR DESCRIPTION
**Pull request by gireeshpunathil**

### CODE REVIEW

**Review by RafaelGSS — COMMENTED** [review]
What do you think about a quick explanation about what is _user journey_?

**Comment by gireeshpunathil** [comment]
@RafaelGSS - done, added a line to that effect, ptal!

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by Qard — APPROVED** [review]

**Review by Flarna — APPROVED** [review]

File: documentation/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# User journey documentation
+Diagnostic user journeys are coherent set of step-by-step procedures,
+that a user follows for problem determination of reported issues.
+The documentation project is organized based on user journey.
+Each user journey is contained in one top level folder by that name.
+
+For each user journey:
+  -  Readme.md contains a high level overview of what use case this flow is addressing
```
Comment (mhdawson) [inline documentation/README.md:8]:
```suggestion
  - Readme.md contains a high level overview of what use case this flow is addressing
```

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by gireeshpunathil** [comment]
landed in420b30ce15b635afce95e55ac487694817e67aab

**Pull request merged**